fleets/day2/chart-templates/elemental/elemental-dashboard-extension/fleet.yaml

```diff
@@ -0,0 +1,9 @@
+defaultNamespace: cattle-ui-plugin-system
+
+helm:
+  releaseName: elemental
```

Comment:
Is this a new component? It's not part of our release notes' list.

Reply:
It is the elemental dashboard extension and was listed in our 3.1 release component spreadsheet. That is why it is added here.